pkg/registry/apis/dashboard/register.go

```diff
@@ -158,7 +159,8 @@ func (b *DashboardsAPIBuilder) GetGroupVersions() []schema.GroupVersion {
 	if featuremgmt.AnyEnabled(b.features, featuremgmt.FlagDashboardNewLayouts) {
 		// If dashboards v2 is enabled, we want to use v2alpha1 as the default API version.
 		return []schema.GroupVersion{
-			dashv2.DashboardResourceInfo.GroupVersion(),
+			dashv2alpha2.DashboardResourceInfo.GroupVersion(),
```

Comment:
@dprokop the API priority works based off of this list – just wanted to check that we want `v2alpha2` as the default one here, is that correct?

Reply:
It's evaluated in the API discovery, so clients like `kubectl` or `grafanactl` will use whatever version the discovery advertises as the default one, which in this PR would be set to `v2alpha2` when the feature toggle is enabled.

If we are concerned about breaking things, I suggest keeping `v2alpha1` as the default version for now, until the conversions are implemented.